db/db_impl.cc

```diff
@@ -4151,7 +4151,7 @@ Status DBImpl::GetDbIdentity(std::string& identity) const {
   if (!s.ok()) {
     return s;
   }
-  char buffer[file_size];
+  char* buffer = reinterpret_cast<char*>(alloca(file_size));
```

Comment:
@igorcanadi I did ran valgrind for their previous version and didn't see any issue. I think this is not new codes.